drivers/misc/coresight/Kconfig

```diff
@@ -73,4 +73,40 @@ config NRF_ETR_SYNC_PERIOD
 	  sent on regular intervals. This frame is sent between Coresight formatter
 	  frames. Use 0 to disable.
 
+config NRF_ETR_SHELL
+	bool "Use shell"
+	select UART_ASYNC_API
+	select UART_ASYNC_RX_HELPER
+	select SHELL_LOG_BACKEND_CUSTOM
+	depends on NRF_ETR_DECODE
+	default y if SHELL
```

Comment:
Will it be a default choice for the shell?

Reply:
yes, when STM logging is used it takes over console UART so when shell is enabled by default it shall cooperate with STM logging - using this backend.